### ISSUE TITLE
Instructor notes out of date

### ISSUE THREAD

**Issue by raybellwaves** [datacarpentry/python-ecology-lesson#307]
Are the instructor notes out of date for the lesson?
http://datacarpentry.org/python-ecology-lesson/guide/

For example, the first few lines in the instructor notes mention how to download python on MacOS using homebrew. Whereas, in the lesson the instructions are to download python using anaconda/miniconda https://datacarpentry.org/python-ecology-lesson/setup.html

**Comment by wrightaprilm** [comment]
You are correct. Merging #305 took care of most of this. I'll handle the installation issues right now, but I'm going to keep this issue open for two things:

- [ ] plotting lesson instructor guide
- [ ] SQL lesson instructor guide

Contributing to either of these would be a fantastic instructor training checkout exercise!

Thanks for noting this - I'm sure a future new instructor looking for a checkout exercise will appreciate it!

**Comment by orchid00** [comment]
hi @wrightaprilm

I found little typos on the guide, section Checking installations
it should refer to _includes/scripts

**Comment by wrightaprilm** [comment]
Oh, thanks for condensing this under an existing issue. I went ahead and committed a fix to this.

**Comment by hot007** [comment]
I'd be happy to have a crack at adding those last two sections to the instructor guide for my checkout process, if that's useful?

**Comment by maxim-belkin** [comment]
That would be great! Thanks, @hot007.

**Comment by grajadiva81** [comment]
Yes, this was a major in a course I took, and the instructor sighted and used a MAC. What happened in the course Mac users and non mac users were separated, Non Mac Users were behind, while Mac Users had a greater advantage to keep up. Non Mac Users were sent into a break out room for a hew hours at a time. However, we could not keep-up. We could listen to the curriculum course instruction, but not the modules. I could not keep up, myself was behind a day 1/2. Would be better to separate users into break out room into mac users and nonusers with two pathways with helpers. would be a instruction adaptation.

**Comment by btovar** [comment]
Fixed in b26d5c8
